sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableSignedIdentifier.java

```diff
@@ -4,12 +4,14 @@
 package com.azure.data.tables.models;
 
 import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 
 import java.util.Objects;
 
 /**
  * A signed identifier.
  */
+@JacksonXmlRootElement(localName = "SignedIdentifier")
```

Comment:
Is this expected?

Reply:
It was part of a bug fixing process but isn't necessary. I'm removing it.